gatsby-config.js

```diff
@@ -45,14 +45,14 @@ module.exports = {
         plugins: [
           `gatsby-remark-responsive-iframe`,
           `gatsby-remark-component`,
-          {
+          /*{
```

Comment:
Do we want to remove this permanently...?

Reply:
Yes! wanted to make sure it was working first. Thanks for reviewing